internal/component/pyroscope/scrape/target.go

```diff
@@ -404,6 +404,9 @@ func targetsFromGroup(group *targetgroup.Group, cfg Arguments, targetTypes map[s
 				lbls = append(lbls, labels.Label{Name: model.AddressLabel, Value: lset.Get(model.AddressLabel)})
 				lbls = append(lbls, labels.Label{Name: model.SchemeLabel, Value: cfg.Scheme})
 				lbls = append(lbls, labels.Label{Name: ProfilePath, Value: lset.Get(ProfilePath)})
+				//ilbls = append(lbls, labels.Label{Name: ProfilePathPrefix, Value: profilePathPrefix})f profilePathPrefix := lset.Get(ProfilePathPrefix); profilePathPrefix != "" {
```

Comment:
Leftover?

Reply:
Yeah. I really want to remove this branch altogether as I believe it is a dead code and never executed